manifests/cf-manifest/manifest/010-cf-jobs.yml

```diff
@@ -8,8 +8,10 @@ meta:
   api_templates:
   - name: consul_agent
     release: (( grab meta.consul_templates.consul_agent.release ))
+    consumes: {consul: nil}
```

Comment:
Out of curiousity, do these Bosh links provide a native solution to the problem that resulted in our creation of https://github.com/alphagov/paas-datadog-for-cloudfoundry-boshrelease ? The problem, if I remember correctly, was the datadog agent trying to access properties from other jobs.